provider/README.md

```diff
@@ -35,6 +35,18 @@ The only 2 required parameters during the set-up are:
 az ad signed-in-user show --query objectId -o tsv
 ```
 
+## Note
+If you would like to recompile a custom version of feast-azure-provider, go to provider/sdk where setup.py is located, run  
+```bash
+python setup.py bdist_egg
+```
+to generate an installable .egg file.
+
+If you would like to install or run a custom version of feast-azure-provider, go to provider/sdk where setup.py is located, run  
+```bash
+python setup.py install
+```
+
 
```

Comment:
Can you elaborate on what a custom version of the feast-azure-provider is?

Reply:
This is to let contributors build and install their own feast-azure-provider for testing. At the moment the docs does not mention how to build and install feast-azure-provider outside of pypi repository - https://pypi.org/project/feast-azure-provider/